book/text.md

```diff
@@ -598,7 +598,7 @@ size is the `<small>` tag and its deprecated sister tag
     we're stuck using them here.
 
 Our experience with font styles and weights suggests a simple
-approach. First, a field in `Layout` to track font size:
+approach that customizes the `size` field in `Layout`. It starts out as:
```

Comment:
The size field was defined above; this text should be about modifying it per-tag.